### ISSUE TITLE
German i18n content is off

### ISSUE THREAD

**Issue by seboslaw** [bluesky-social/social-app#2316]
**Describe the bug**

I saw that the app added German i18n translations. However, it seems that at least some of them are done through an AI translation service - they're so terribly off that it makes the app look suspicious. Also, e.g. in the Home Feed Preferences some of the strings are still in English. so it's mixed and confusing.

**To Reproduce**

1. Set your iOS system language to German.
2. open the app and open the Home Feed Preferences

**Expected behavior**

I expect all strings to be translated in comprehensible german language.

**Screenshots**

![IMG_7B0B2E3AB253-1](https://github.com/bluesky-social/social-app/assets/97770/5506332c-c4c2-4b8d-8d7e-3461593f4d51)

**Details**

 - Platform: iOS 
 - Platform version: 17.1.2
 - App version: 1.61.0 (3)

**Comment by imbstt** [comment]
Agreed, I posted some feedback in an old issue here https://github.com/bluesky-social/social-app/issues/1720#issuecomment-1869761854
